tests/test_kubernetes_logs.py

```
@@ -0,0 +1,245 @@
+    assert len(result) == 3
+    assert count == 10
+
+
+def test_filter_logs_invalid_regex_falls_back_to_substring():
+    logs = [
+        _make_log("CrashLoopBackOff: back-off restarting"),
+        _make_log("info: all good"),
+    ]
+    # "[invalid" is not valid regex — should fall back to substring match
+    params = _make_params(filter="[CrashLoop")
+    result, _, used_fallback, *_ = filter_logs(logs, params)
```

Ruff (0.15.18)

[warning] Unpacked variable `result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)